int32 left_speed
int32 right_speed
bool trash_full
bool emergency
int32 left_encoder
int32 right_encoder